index.html

```diff
@@ -90,7 +90,7 @@ <h2 class="support-text-2 default-custom-fonts lets-text-font">LETS CONNECT</h2>
           <li><img class="counter" src="images/Counter.svg" alt="Counter"></li>
```

Comment:
- [ ] Fix margin/padding in this section.
- [ ] Also you can add the right background-color for the project technologies list "HTML", "css", "javascript".

Figma Design | Yours

![image](https://user-images.githubusercontent.com/60164239/129461378-bc35a934-969a-495b-afcf-86c5de18538f.png)